index.html

```diff
@@ -2,114 +2,114 @@
-
-          <div class="abc__second">
-            <ul class="fry">
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#welcome">Welcome</a>
-
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#about">About</a>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="top-nav">
+          <h1 class="top-nav__left">Shia LaBeouf</h1>
+
+          <div class="top-nav__left">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#welcome">Welcome</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#about">About</a>
               </li>
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#catchphrases">Quotes</a>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#quotes">Quotes</a>
               </li
-              <li class="fry__thing">
-                <a class="fry__zelda" href="#photos">Photos</a>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#photos">Photos</a>
               </li>
             </ul>
 
         <div>
       </div>
     </nav>
-    <div class="bender__content">
-      <section class="area-of-control area-of-control--zapp" id="welcome">
-        <div class="box>
-          <div class="zapp">
-            <div class="zapp__thing">
-              <h2 class="zapp__text">This is something interesting about me.</h2>
+    <div class="app__content">
+      <section class="section section--hero" id="welcome">
+        <div class="container">
+          <div class="hero">
+            <div class="hero__item">
+              <h2 class="hero__text">This is something interesting about me.</h2>
             </div>
-            <div class="zapp__thing">
-              <img src="http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg" alt="" class="zapp__img">
+            <div class="hero__item">
+              <img src="http://www.bobbyberberyan.com/wp-content/uploads/2012/03/HTML5CSS3Logos.svg" alt="" class="hero__img" />
             </div>
           </div>
         </div>
-      </area-of-control>
-
-      <div class="area-of-control">
-        <div class="box">
-          <h2 class="area-of-control__heading">I'm an interesting person!</h2>
-
-          <div class="leela">
-            <div class="leela__thing">
-              <div class="blackjack">
-                <div class="twitter">
-                  <div class="twitter__first">
-                    span class="morty morty--blue">
+      </section>
+
+      <div class="section">
+        <div class="container">
+          <h2 class="section__heading">I'm an interesting person!</h2>
+
+          <div class="grid">
+            <div class="grid__item">
+              <div class="card">
```

Comment:
From opening this in the browser, it looks like maybe this card is missing a closing tag somewhere?